formulas/wick-base/explorers/os

```diff
@@ -62,6 +62,19 @@ if [[ -f /etc/arch-release ]]; then
     exit
 fi
 
+# Check os-release files for ID
+if [[ -f /etc/os-release ]]; then
+    . /etc/os-release
```

Comment:
Grep is fine.  FWIW though, you'd have to be root to write to either location.  I'm hard pressed to think of an instance in which someone could do something malicious without already having the "keys to the kingdom".  The only thing sourcing the file gives us is environment variables that we could, in theory, use later.

I'll change to grep(1).